src/common/no-deps.ts

```diff
@@ -99,22 +76,21 @@ export enum LOG_LEVELS {
   ERROR = -1,
   WARN = 0,
   INFO = 1,
-  VERBOSE = 1,
   DEBUG = 2,
   TRACE = 3,
 }
 
 let logLevel = Number.NEGATIVE_INFINITY
 export const LOG_LEVEL = (): number => {
-  if (!ENV.PARSED_ARGS) return LOG_LEVELS.ERROR
+  if (!getParsedArgs()) return LOG_LEVELS.ERROR
   if (logLevel > Number.NEGATIVE_INFINITY) return logLevel
 
-  let LL = Number(LOG_LEVELS[(ENV.PARSED_ARGS as BasicArguments).logLevel?.toUpperCase() ?? 'WARN'])
-  const verbosity = Number((ENV.PARSED_ARGS as BasicArguments).verbose ?? 0)
-  const boolTrace = cleanedEnv.TRACE || ENV.PARSED_ARGS.trace
-  LL = boolTrace ? LOG_LEVELS.TRACE : LL
+  let logLevelNum = Number(LOG_LEVELS[getParsedArgs().logLevel?.toUpperCase() ?? 'WARN'])
```

Comment:
please only use uppercase for constants, so define `LOG_LEVELS` as `logLevels`

Reply:
Sure, but aren't enums basically constants?